src/components/Gallery.js

```diff
@@ -23,11 +24,10 @@ class Gallery extends Component {
     }
 
     changeImage() {
-        if (this.state.imageIndex === this.props.images.length - 1) {
-            this.setState({ imageIndex: 0 });
-        } else {
-            this.setState({ imageIndex: this.state.imageIndex + 1 });
-        }
+        this.setState({ 
+            imageIndex: (this.state.imageIndex === this.props.images.length - 1) ? 
+                0 : (this.state.imageIndex + 1) 
+        });
```

Comment:
KISS. Try to write the same logic but with less lines of code. At the same time, try to make the code easy to understand: clear indentation can help with that too.